### PR TITLE
Update for 1.1.1/1.0.4 and remove project.json images

### LINKED ISSUE #200: Drop support for the project.json images
At some point in the near future, we will stop supporting the project.json images. When we do that, we can remove Dockerfiles and simplify test code.

cref #199

### LINKED ISSUE #203: Produce 1.0.4 and 1.1.1 images
Prepare latest patch update for release.

### PR DESCRIPTION
**Pull request by natemcmaster**
Update packagecache to use the "Feburary 2017" patch release
Update crossgen caches
Update README.md examples.

Resolves #203 
Resolves #200

### CODE REVIEW

File: 1.1/jessie/sdk/Dockerfile [hunk 1]
```diff
@@ -1,11 +1,11 @@
-FROM microsoft/dotnet:1.1.0-sdk-msbuild-rc4
+FROM microsoft/dotnet-nightly:1.1.1-sdk
```
Comment (natemcmaster) [inline 1.1/jessie/sdk/Dockerfile]: @MichaelSimons 

```
Step 1/8 : FROM microsoft/dotnet-nightly:1.1.1-sdk
manifest for microsoft/dotnet-nightly:1.1.1-sdk not found
```

**Comment by BrennanConroy** [comment]
Shouldn't it be patch 1.1.2

File: 1.1/jessie/sdk/Dockerfile (hunk 1 above)
Comment (natemcmaster) [inline 1.1/jessie/sdk/Dockerfile]: Any ideas? This tag is listed on docker hub.

**Comment by natemcmaster** [comment]
@BrennanConroy good catch. The package cache version is "1.1.2", but we're keeping the image versions aligned with the dotnet images and the runtime version.

File: 1.1/jessie/sdk/Dockerfile (hunk 1 above)
Comment (MichaelSimons) [inline 1.1/jessie/sdk/Dockerfile]: Docker Hub is current down at the moment -  http://status.docker.com/

Comment (natemcmaster) [inline 1.1/jessie/sdk/Dockerfile]: Oh, thx 🤒

File: 1.1/jessie/sdk/README.md [hunk 2]
```diff
@@ -60,7 +56,7 @@ With this technique your application is compiled when you run `docker build` and
     FROM microsoft/aspnetcore-build
     WORKDIR /app
 
-    COPY project.json .
+    COPY MyApp.csproj .
```
Comment (MichaelSimons) [inline 1.1/jessie/sdk/README.md]: Not sure what I think about this, but would it make sense to use `*.csproj` here as a way to abstract the Dockerfile?

Comment (natemcmaster) [inline 1.1/jessie/sdk/README.md]: Yeah, good idea. Makes it more copy+paste friendly too.

**Comment by natemcmaster** [comment]
:watch: https://github.com/dotnet/dotnet-docker/pull/214

**Pull request merged**